README.md

```diff
@@ -1747,6 +1747,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Tiledesk](https://tiledesk.com) - Create chatbots and conversational apps. Bring them omnichannel: from your website (live chat widget) to WhatsApp. Free plan with unlimited chatbots.
   * [Versionfeeds](https://versionfeeds.com) — Custom RSS feeds for releases of your favorite software. Have the latest versions of your programming languages, libraries or loved tools in one feed. (First 3 feeds are free)
   * [videoinu](https://videoinu.com) — Create and edit screen recordings and other videos online.
+  * [Legally Blocked](https://www.legallyblocked.website/) - Find Blocked website, mobile apps and books country wise, Free for all. Feedback Welcome
```

Comment:
I guess its useful, but I dont see the relevance to this list sorry.

Reply:
This is a free project with free API, as per me this can be used by the dev to find alternate if there app is using any 3rd party services. (May be what's app API or some other public API which is blocked in certain regions)

Reply:
I understand. There are many free project not all listed here. This list has a purpose and I don’t think your site fits for the purpose of this list. Read top of readme. 


thanks you.